content/blog/federating-spire-on-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,441 @@
+```
+
+![Cluster-1 Logs](/img/table3.png)
+
+```shellsession
+Cluster2:~ # kubectl logs -n spire -c spire-server spire-server-574474c7dc-2bfcx 
+```
+
+![Cluster-2 Logs](/img/table4.png)
+
+# Step 4. Deploying a sample application :
+
+Now that SPIRE is federated and communication across clusters can be facilitated, let us deploy a sample application that verifies this functionality.   
+
+## 4.1 Deploy a resource in Cluster-1
+
+In Cluster 1, apply a new ClusterSpiffeID called federated that registers resources with the label **spiffe.io/spire-managed-identity=curl-greeter** that can be federated with cluster2. Create a resource called curl-greeter that has the label: **spiffe.io/spire-managed-identity=curl-greeter** and annotation: **inject.istio.io/templates=sidecar, spire** 
+
+```shellsession
+kubectl run curl-greeter --image=radial/busyboxplus:curl --labels="spiffe.io/spire-managed-identity=curl-greeter" --overrides='{ "apiVersion": "v1", "spec": { "template": {"metadata": {"annotations": { "inject.istio.io/templates":"sidecar,spire" } } }}}' -i --tty 
+```
+
+## 4﻿.2 Deploy Bookinfo Sample Application in Cluster-2
+
+In Cluster 2, apply a new ClusterSpiffeID called federated that registers resources with the label **spiffe.io/spire-managed-identity=spire** that can be federated with cluster1. Apply the bookinfo sample application manifest. 
+
+```shellsession
+kubectl apply -f services/istio/release-1.17/bookinfo.yaml 
+```
+
+## 4﻿.3 Check if all the resources created are up and running:
+
+```shellsession
+Cluster1:~ # kubectl get po 
+
+NAME                              READY   STATUS    RESTARTS   AGE 
+
+curl-greeter                      2/2     Running   0          15h 
+```
+
+```shellsession
+Cluster2:~ # kubectl get po 
+
+NAME                                READY   STATUS    RESTARTS   AGE 
+details-v1-bff8759df-vkvb4          2/2     Running   0          16h 
+greeter-client-76686757cd-6j2ft     2/2     Running   0          21h 
+productpage-v1-98887b9b-x5k24       2/2     Running   0          16h 
+ratings-v1-7ddbb859fc-htmfq         2/2     Running   0          16h 
+reviews-v1-67b576c8bf-jr6tj         2/2     Running   0          16h 
+reviews-v2-7ffbdcc5f7-m2c29         2/2     Running   0          16h 
+reviews-v3-6dbfcc6d89-zn9tw         2/2     Running   0          16h 
+```
+
+## 4﻿.4 Check that the workload identity was issued by SPIRE
+
+Spire Certificate at Curl-Greeter 
+
+```shellsession
+Cluster1:~ # istioctl proxy-config secret curl-greeter -o json | jq -r \ 
+
+> '.dynamicActiveSecrets[0].secret.tlsCertificate.certificateChain.inlineBytes' | base64 --decode > chain.pem 
+
+Cluster1:~ # openssl x509 -in chain.pem -text | grep SPIRE 
+
+        Subject: C = US, O = SPIRE, x500UniqueIdentifier = c3dd29b5f4a326f8a567854407456ea9 
+```
+
+*Note: Similarly, certificates for other pods can be checked.* 
+
+## 4﻿.5 Create a secret to hold bookinfo product page credentials like this: 
```

Comment:
This is an instance similar to the one shown earlier where this appears to be the instructions that you are giving the reader as opposed to an actual header. I'd recommend creating an actual header first that says something like "Create a secret" and then place this, in regular font, as a sentence below it. This way you maintain a consistent use of 1) header 2) instructions.